package.json

```diff
@@ -39,6 +39,7 @@
     "@metamask/eslint-config-jest": "^9.0.0",
     "@metamask/eslint-config-nodejs": "^9.0.0",
     "@metamask/eslint-config-typescript": "^9.0.1",
+    "@metamask/eth-json-rpc-provider": "^1.0.0",
```

Comment:
Do these need to be added to the root manifest?

Reply:
Yes. I moved FakeBlockTracker from the `network-controller` package to the root `tests/` directory so that I could access it in the TransactionController tests. It would probably still work if I kept it in `network-controller`, however, to import FakeBlockTracker I would have to use a relative path (`import { FakeBlockTracker } from "../../../network-controller/tests/fake-block-tracker"` rather than `import { FakeBlockTracker } from "@metamask/network-controller"`) and I don't want to encourage that, but I guess an exception could be made here if it makes more sense. But that's why.

Reply:
Gotcha; I hadn't noticed that was moved. This makes sense then.